### DISCUSSION TITLE
特定の行と列のキーが反応しない

### DISCUSSION THREAD

**Discussion by tyamap** [Yowkees/keyball#370] [Q&A]
お世話になっております。
keyball39 にて、画像で赤枠で示したキー（Y行とP列）が全て反応しません。
原因となりうる部分やテスト方法をご教示いただけますでしょうか？

下記備考です。

- 過去にP列が反応しない症状があり、ProMicroのはんだやダイオード、キーソケットのはんだをやり直し、改善しました
- その後、全てのキーが動作しているのを確認して実際に使用していました
- 1日ほど使用していたところ、突然反応しなくなりました
- 右側の ProMicro に差して使用していました
- どちらの ProMicro に差しても反応しません
- 何度かUSBを抜き差ししても反応しません

どうぞよろしくお願いします。

![IMG_6826 2](https://github.com/Yowkees/keyball/assets/43022176/29a4f52a-d009-459a-aeb2-16388beb69b8)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

本件に該当するのが、ProMicro裏でCOL0とROW0の刻印がある端子になります。
画像を確認しましたが、隣とショートしているということはなさそうですね。

ですので、ProMicro自体の半田付けか、12Pピンソケットの導通が怪しいと思います。
左右のProMicroを入れ替えてみるとどうなるでしょうか？

また、ミドル基板とProMicro間の該当端子が導通しているかテスターをお持ちでしたらご確認いただければ幸いです。

以上、お手数ですがご確認お願い足します。

**Reply by tyamap** [reply]
ご返信いただきありがとうございます。
左右の ProMicro を入れ替えたところ、正常に動作しました。
ご教示いただいた通り、12Pピンソケットとの導通が不十分だったようです。該当のピンを半田付けし直したところ、正常に動作するようになりました。

ご対応いただきありがとうございます🙌

**Comment by Yowkees** [comment]
無事に動くようになったようで安心しました。
ぜひご活用いただければ幸いです。
